.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -6,44 +6,30 @@ labels: bug
 assignees: ''
 
 ---
-
-[TIP]:  # ( Provide a general summary of the issue in the title above ^^ )
+## Overview
 [TIP]:  # ( DO NOT include screenshots of your actual database! )
+[NOTE]: # ( Give a BRIEF summary about your problem )
+
+
+## Steps to Reproduce
+[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
+1. 
+2. 
+3. 
 
 ## Expected Behavior
 [NOTE]: # ( Tell us what you expected to happen )
 
 
-## Current Behavior
+## Actual Behavior
 [NOTE]: # ( Tell us what actually happens )
 
 
-## Possible Solution
-[NOTE]: # ( Not required, but suggest a fix/reason for the bug )
-
-
-## Steps to Reproduce
-[NOTE]: # ( Provide a link to a live example, or an unambiguous set of steps to )
-[NOTE]: # ( reproduce this bug. Include code to reproduce, if relevant )
-1.
-2.
-3.
-
 ## Context
-[NOTE]: # ( How has this issue affected you? What unique circumstances do you have? )
+[NOTE]: # ( Give us any additional information you may have. )
 
 
-## Debug Info
 [NOTE]: # ( Paste debug info from Help → About here )
 KeePassXC - VERSION
 Revision: REVISION
 
```

Comment:
I would suggest having at least the OS in there as well or nothing at all. KeePassXC version alone isn't enough in almost all cases, whereas a note of the OS may be sufficient to help some already without repeatedly asking them to provide the full debug info.

Reply:
Good point, and desktop env for Linux